src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -219,7 +219,7 @@ void goto_analyzer_parse_optionst::get_command_line_options(optionst &options)
     options.set_option("intervals", true);
     options.set_option("domain set", true);
   }
-  else if(cmdline.isset("(show-non-null)"))
+  else if(cmdline.isset("show-non-null"))
```

Comment:
Anything to do with the PR? If not submit this on its own

Reply:
It's not but it's a 1-line change on a broken feature which I noticed while editing nearby code so I thought it would be just less hassle to fit it in here.